### ISSUE TITLE
[TalkBack] Allow user to specify how to speak punctuation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#252]
```
One should be able to adjust the level of punctuation read by Talkback. This 
would be useful in that it would allow easier digestion of things such as code, 
or things such as proof-reading messages.

What version of the product are you using?
Talkback 3.3.2
On what operating system?
Android 4.2.2
```

Original issue reported on code.google.com by `thomasc...@gmail.com` on 27 Apr 2013 at 11:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 30 Apr 2013 at 7:44
- Changed title: **[TalkBack] Allow user to specify how to speak punctuation**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**
